src/components/HeartbeatBar.vue

```diff
@@ -14,7 +14,7 @@
             v-if="!$root.isMobile && size !== 'small' && beatList.length > 4 && $root.styleElapsedTime !== 'none'"
             class="d-flex justify-content-between align-items-center word" :style="timeStyle"
         >
-            <div>{{ timeSinceFirstBeat }} ago</div>
+            <div>{{ $t("ago", [timeSinceFirstBeat]) }}</div>
```

Comment:
I don't think this is a good translation key (would you as a translator know what to put there without additional context?)
unsure about better keys though. Proposal: "time ago"

Reply:
It would be necessary to change the key to "heartbeatTimeAgoMSG," for example, and add this screenshot to Weblate to enable the translator to understand what it is about.

![image](https://github.com/louislam/uptime-kuma/assets/70776486/8030bfb3-aeef-49b6-95ef-3a9ba6f59d8a)

Reply:
I don't think we need to restrict usage to heartbeats via the key
⇒ I think `time ago` would be a better key.
Agree that the screenshot would be helpful.